onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -559,6 +568,9 @@ def testInputObject(self):
         np.testing.assert_equal(x, res[0])
 
     def testInputVoid(self):
+        if 'TvmExecutionProvider' in available_providers:
+            print('Skipped. TVM EP doesn\'t support void type inputs.')
```

Comment:
I would prefer a comment rather than printing a message or maybe print it only once.

Reply:
Ok, I'll use the comment. Print was made by analogy with [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L1025), [DML](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L392).